### ISSUE TITLE
Status, licensing of this package

### ISSUE THREAD

**Issue by rehno-lindeque** [mokus0/ecma-048#1]
I noticed that this package looks superficially quite promising for my purposes.

However, it's hard to judge the maturity and origin of this package at the outset. Would you consider clarifying the license and publishing to hackage? How complete is it (or, how much work does it need?)

**Comment by rehno-lindeque** [comment]
Actually, I see that https://github.com/mokus0/ecma-048/blob/12e81bbbe300a025fbe1c864308b0aeb4d9bf145/ecma-048.cabal#L10 lists this as public domain?

**Comment by mokus0** [comment]
Yes, my intention was to claim no rights on this.  I don't remember how much work actually went into it, but as I recall it's pretty minimal.  I don't have the available time these days to manage any projects, but if you want to take it and run with it, go for it.

**Comment by rehno-lindeque** [comment]
I ended up starting from scratch because I decided that I wanted a `ParsecT` parser for my use case https://github.com/rehno-lindeque/ansi-megaparsec (very much experimental and WIP). Thanks though!
